src/ng/http.js

```diff
@@ -1088,7 +1094,12 @@ function $HttpProvider() {
         }
 
         // send request
-        return sendReq(config, reqData).then(transformResponse, transformResponse);
+        try {
+          var promise = sendReq(config, reqData);
```

Comment:
nit: I find it a bit more readable to define `var`s in the block scope that they are used in, even if they are function scoped. That is, move the declaration of `var promise` up one line?